samples/maps/series-geoheatmap/turkey-fire-areas/demo.js

```diff
@@ -0,0 +1,709 @@
+(async () => {
+
+    const topology = await fetch(
+        'https://code.highcharts.com/mapdata/custom/world.topo.json'
+    ).then(response => response.json());
```

Comment:
Is this used at all? This demo has only a twm and a geoheatmap series, so we probably shouldn't load the topoJSON.

Reply:
Yes, it is used to create outlines of continents.